tests/foreman/api/test_activationkey.py

```diff
@@ -42,26 +39,12 @@ def test_positive_create_2(self, max_content_hosts):
         @Feature: ActivationKey
 
         """
-        try:
-            attrs = entities.ActivationKey(
-                unlimited_content_hosts=False,
-                max_content_hosts=max_content_hosts,
-            ).create_json()
-        except HTTPError as err:
-            self.fail(err)
-        # Assert that it defaults to limited content host...
-        self.assertFalse(
-            attrs['unlimited_content_hosts'],
-            u"Unlimited content hosts is {0}".format(
-                attrs['unlimited_content_hosts'])
-        )
-        # ...and matches value passed
-        self.assertEqual(
-            attrs['max_content_hosts'],
-            max_content_hosts,
-            u"Max content hosts values don't match: {0} != {1}".format(
-                attrs['max_content_hosts'], max_content_hosts)
-        )
+        act_key = entities.ActivationKey(
+            max_content_hosts=max_content_hosts,
+            unlimited_content_hosts=False,
+        ).create()
```

Comment:
Both will make the test fail, but with the try/catch it will increase the fail number and without the error number. For results it does not matter at all.

We have better logging and other stuff to help identify it that is a robottelo bug or a server fault.